### DISCUSSION TITLE
Streaming-only mode?

### DISCUSSION THREAD

**Discussion by Tzahi12345** [Tzahi12345/YoutubeDL-Material#666] [Polls]
Streaming-only mode was a per-subscription setting that meant no videos would get downloaded. Simply the streaming URL would be retained. It was basically a way to view your subs without having to go to the host website (e.g. youtube.com).

In v4.3 we are going to remove the feature as it was previously quietly deprecated, but it appears it has been discussed in the past (#83) and I thought I'd make this poll to gauge interest.

**Comment by GlassedSilver** [comment]
I think if it's not too much effort it could be worthwhile to re-integrate that feature properly, since it seems like a valid USP for folks who want to watch YT and other content without logging in and personally hosted. Basically you can make the point that even without having to download everything, YTDL-M can be your personal alternative front-end for YT AND other sites at the same time - while other YTDL GUIs - I think - always force you to download videos.

If you want to keep up with everything through YTDL-M and only archive your favorites, I think this function proves incredibly useful, and maybe we just haven't communicated this feature enough (yet) to attract a user base that may not even have considered us yet or didn't know that there is a solution to mixing archiving + consumption without downloads.

If you ask me, I wouldn't mind importing all of my YT subs, maybe with a very decreased check interval for the bulk and more frequent for the stuff I actually want to archive and convert a streaming-only sub to a full one à la carte at any point when I feel that either the channel is on the way of getting deleted or I started liking it a lot more.

Certainly this could be useful for those times when YT's homepage algorithm just isn't vibing with your current mood too much and YTDL-M might bring a better recommendation algorithm at some point. Personally I see a lot of potential in this feature for it to not get axed so quickly,

**Comment by rockbo842** [comment]
I didn't understand what the feature was for but @GlassedSilver makes a good point.  If it isn't a huge deal to reincorporate it, I think it would be a good idea.
